packages/content-format/src/traits/AlignableBlock.ts

```diff
@@ -0,0 +1,29 @@
+import { isEnum } from '../validation';
+
+export type AlignableBlock<T extends Record<string, any>, Required extends boolean = true> = T &
+    (Required extends true ? { align: `${AlignableBlock.Alignment}` } : { align?: `${AlignableBlock.Alignment}` });
+
+export namespace AlignableBlock {
+    export enum Alignment {
+        LEFT = 'left',
+        CENTER = 'center',
+        RIGHT = 'right',
+    }
+
+    export function validateAlignable<T extends Record<string, any>, Required extends boolean = true>(
```

Comment:
Was wondering if I should rename this to `validateAlignableBlock` (to reflect the type name) or just call it `validate` (since it already exists within a namespace).

Reply:
While `validate()` does make sense indeed, I'd go with `validateAlignableBlock` to maintain consistency with everything else.